GOVERNANCE.md

```diff
@@ -1,53 +1,80 @@
-The guiding approach for ensuring this is the one of [lazy consensus](https://community.apache.org/committers/decisionMaking.html).
+The Open Quantum Safe project aims to operate by the following principles:
 
-## Fallback
+- **Openness**: The project is will be open in its operation, open to contributions, and produce open source software.
+- **Respect**: The project will foster respectful interactions with all participants and contributors.
+- **Scientific integrity**: The project will be follow advancements in cryptographic research and will be guided by standards and best practices.
 
-This file documents the formal governance guidelines used for this project. While it also clarifies roles, responsibilities and expectations to contributors at different levels, it is of most relevance only in situations if decisions cannot be reached using lazy consensus but a more "formal" approach to decision-making is needed.
+Decision making in the project will follow the principles above, and be governed first and foremost by reason and mutually respectful interaction between all contributors.
+The project will aim to build consensus for decisions, and will where possible operate by the approach of [lazy consensus](https://community.apache.org/committers/decisionMaking.html).
+If decisions cannot be reached using lazy consensus, voting will be used to come to a resolution.
 
-The following therefore defines the roles of project contributors, the associated rights and responsibilities, and the process for transitioning between them. As such, this document is written in a fairly formal and precise tone, so as to be succint and unambiguous. This should not be interpreted as a lack of warmth on the part of the OQS team---we're really quite friendly! We do not intend to act as gatekeepers by laying out this tier of roles and the associated rules. Instead, we hope that clearly defining these roles and the processes for attaining them shows contributors a clear path by which to become more involved in project governance, if they so wish. We welcome all questions, discussions, and contributions, and we would love to have more people on board.
+## Community and Roles
 
-We recognize that some of the policies discussed here can seem intimidating---for instance, revocation of privileges or code of conduct violations. It is our hope that we don't have to rely on these guidelines; however, we believe that it is important to have them in place should they be needed.
-
-## Roles
+The OQS community is open to all who would like to participate in the project following its principles, including contributors in academia, government, or industry, and individual contributors.
 
 The following roles exist in the project:
 
-1. Maintainer: Person with GitHub administrative rights.
+### Users
+
+A **User** is a person or organization using software produced by the project.
+
```

Comment:
What about adding a "Responsibility" section here, too? What comes to mind is 
- abiding by the license
- at least considering to "give back"
I'd be particularly glad to have at least one place where "take-only" users (typically profit-minded companies) are reminded that FOSS cannot only be a one-way road in their favour.